### ISSUE TITLE
Touchpad working but not very well (HP ProBook x360 11 G1 EE)

### ISSUE THREAD

**Issue by Maxcension** [sebanc/brunch#648]
Hello!

I have an HP ProBook x360 11 G1 EE but I have a problem with the touchpad (an Elan one).
The thing is that the touchpad works, but really really bad, like there are some areas of the touchpad that is simply not working (and these areas works great on windows or in the bios).
I tried the ""alt_touchpad_config" / "alt_touchpad_config2"" in the grub config but its but it doesn't change so much.
I'm using the snappy recovery image (because the HP ProBook x360 11 G1 EE also exists with chromeos).

Thanks a lot! 👍

**Comment by Maxcension** [comment]
So, i tried some things like "psmouse.elantech_smbus=1" refered in #623, use "psmouse proto=imps" thing or creating file /etc/modprobe.d/elan.conf with "elantech_smbus=0 synaptics_intertouch=0" ( https://bbs.archlinux.org/viewtopic.php?pid=1861098#p1861098 ) but none of these really worked.

So yeah i'm pretty stuck, I think i have to modify the /etc/gesture/40-touchpad_cmt.conf but i dont know what to do exactly..

**Comment by jafer241177** [comment]
same problem  in lenovo idealpad flex 3  but no solve   for this big problem no update driver in chrome  all out in bios

**Comment by Maxcension** [comment]
Yeah, i think we have to wait for someone with the same computer or a close one and who will have better knowledge than us. :(
Personnally i tried everything i could but i didn't succeed. :/

If anyone comes by and has a solution or plans to help us, it would be greatly appreciated! Thank you very much.

**Comment by klutzyT** [comment]
i have the same computer (hp probook x360 11 g1 ee) and i used a coral image on r103 and originally the touch pad was just as you had mentioned sticky and just terrible so i used psmouse.elantech_smbus=1 kernel command line and it fixed itself but now the speakers dont work on here does anyone have a fix for that?
